image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -65,6 +65,10 @@ spec:
           value: {{ ._rox.sensor.endpoint }}
         - name: SNI_HOSTNAME
           value: "sensor.stackrox.svc"
+        [<- if .FeatureFlags.ROX_COLLECTOR_RUNTIME_CONFIG >]
+        - name: ROX_COLLECTOR_RUNTIME_FILTERS_ENABLED
+          value: "true"
```

Comment:
Should it be allowed that users of the Helm chart (note, the operator is also "user" of the Helm chart) overwrite this option?

Reply:
Good call, I'll add a variable.